alonzo/impl/src/Cardano/Ledger/Alonzo/Translation.hs

```diff
@@ -112,6 +121,14 @@ instance Crypto c => TranslateEra (AlonzoEra c) ShelleyGenesis where
 
 newtype TxInBlock era = TxInBlock (Era.TxInBlock era)
 
+deriving newtype instance
```

Comment:
Remind me what we need this instance for? Just testing?

Reply:
It is. I'll move it.